ansible/roles/debops.apt/defaults/main.yml

```diff
@@ -485,8 +524,10 @@ apt__security_sources:
 apt__combined_sources: '{{ apt__sources +
                            apt__group_sources +
                            apt__host_sources +
-                           apt__original_sources +
-                           apt__default_sources +
+                           apt__original_sources }} +
+                           {{ apt__default_sources if apt__default_sources_present else [] }} +
+                           {{ apt__host_security_sources +
+                           apt__group_security_sources +
```

Comment:
This is an unexpected syntax. The entire value should be enclosed in one pair of `{{ }}` parentheses, basically something like this:
```
apt__combined_sources: '{{ apt__sources +
                           ...
                           apt__original_sources +
                           (apt__default_sources if (apt__default_sources_present|bool) else []) +
                           apt__host_security_sources +
                           ...
                           apt__security_sources }}'
```
Additionally, why didn't you set the status of APT default sources similarly to how you defined it in APT security sources? Both methods seem feasible to do the same thing, there keeping both configurations the same would make things consistent.

Reply:
Ah, that's the syntax I was after. Thanks!

The reason I put this logic here in the first place was because I didn't want to mess with the local fact `default_sources_map`. I noticed this local fact is used in debops.apt_cacher_ng, and should always contain the  mirrors from `apt__default_sources`.
Allowing change of the 'state' of the items in `apt__default_sources` to `absent` would cause `ansible_local.apt.default_sources_map` to be empty. Which probably should be avoided.